<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>381</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>381</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ORIGINAL TEXT:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>CIPHER:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>CIPHER</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>458</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuIle">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionSave_as_cipher_text"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionInfo"/>
   </widget>
   <widget class="QMenu" name="menuJames_Bond">
    <property name="title">
     <string>James Bond</string>
    </property>
    <addaction name="actionCiphier"/>
   </widget>
   <addaction name="menuIle"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
   <addaction name="menuJames_Bond"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_as"/>
   <addaction name="actionSave_as_cipher_text"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="actionCut"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionInfo"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/images/C:/Users/User/Desktop/images/new.png</normaloff>:/images/C:/Users/User/Desktop/images/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/images/C:/Users/User/Desktop/images/open.png</normaloff>:/images/C:/Users/User/Desktop/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/images/C:/Users/User/Desktop/images/save.png</normaloff>:/images/C:/Users/User/Desktop/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/images/C:/Users/User/Desktop/images/Save-as.png</normaloff>:/images/C:/Users/User/Desktop/images/Save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save as normal text</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/images/C:/Users/User/Desktop/images/copy.png</normaloff>:/images/C:/Users/User/Desktop/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/images/C:/Users/User/Desktop/images/cut.png</normaloff>:/images/C:/Users/User/Desktop/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/images/C:/Users/User/Desktop/images/paste.png</normaloff>:/images/C:/Users/User/Desktop/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/images/C:/Users/User/Desktop/images/Undo.png</normaloff>:/images/C:/Users/User/Desktop/images/Undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/images/C:/Users/User/Desktop/images/Redo.png</normaloff>:/images/C:/Users/User/Desktop/images/Redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/images/C:/Users/User/Desktop/images/About.png</normaloff>:/images/C:/Users/User/Desktop/images/About.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
  </action>
  <action name="actionCiphier">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/images/C:/Users/User/Desktop/images/bond.png</normaloff>:/images/C:/Users/User/Desktop/images/bond.png</iconset>
   </property>
   <property name="text">
    <string>Cipher</string>
   </property>
  </action>
  <action name="actionSave_as_cipher_text">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/images/C:/Users/User/Desktop/images/images.png</normaloff>:/images/C:/Users/User/Desktop/images/images.png</iconset>
   </property>
   <property name="text">
    <string>Save as cipher text</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>


